gateway/1.4/modules/ROOT/pages/flex-gateway-getting-started-kubernetes.adoc

```diff
@@ -67,7 +63,7 @@ The following minimum requirements apply to a small Kubernetes cluster used for
 
 * Support for the creation of a `Service` resource of the `LoadBalancer` type.
 +
-To avoid deployment errors when using this guide, your cluster must support this process. However, Flex Gateway supports these Ingress Controller services: `ClusterIP`, `NodePort`, `LoadBalancer` (default), `ExternalName`. 
+This requirement avoids deployment errors when you use this guide. However, it is possible to customize the `Service` type when deploying a gateway. Flex Gateway supports the following services: `ClusterIP`, `NodePort`, `LoadBalancer` (default), `ExternalName`. See the `service.type` key in the https://artifacthub.io/packages/helm/flex-gateway/flex-gateway[Helm chart for Flex Gateway^].
```

Comment:
Is the "when you use this guide" necessary?

Reply:
Yes, it's necessary only for the procedures in this guide. It would not avoid errors if they used another valid configuration. The procedures here are but one possible configuration.